src/ck-ldb/MetisLB.C

```diff
@@ -5,128 +5,126 @@
 
 /**
  * \addtogroup CkLdb
-*/
+ */
 /*@{*/
 
 #include "MetisLB.h"
 #include "ckgraph.h"
+#include <cstddef>
 #include <metis.h>
 
 extern int quietModeRequested;
 
 CreateLBFunc_Def(MetisLB, "Use Metis(tm) to partition object graph")
 
-MetisLB::MetisLB(const CkLBOptions &opt): CBase_MetisLB(opt)
+    MetisLB::MetisLB(const CkLBOptions& opt)
+    : CBase_MetisLB(opt)
 {
   lbname = "MetisLB";
-  if (CkMyPe() == 0 && !quietModeRequested)
-    CkPrintf("CharmLB> MetisLB created.\n");
+  if (CkMyPe() == 0 && !quietModeRequested) CkPrintf("CharmLB> MetisLB created.\n");
```

Comment:
fwiw, I prefer the old way (with a line break before the `CkPrintf`).

Reply:
Fixed.

Reply:
If this a thing we want to enforce, we should set `AllowShortIfStatementsOnASingleLine` in the Clang-Format configuration correspondingly.

Reply:
Done in #3229.